src/components/OrgCard/index.jsx

```diff
@@ -1,11 +1,61 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import "./style.css";
+function OrgCard() {
+  const [selectedIndex, setSelectedIndex] = useState(null);
+  let bgColors = [
+    'rgba(105, 196, 105, 1)',
+    'rgba(247, 211, 68, 1)',
+    'rgba(122, 189, 244, 1)'
+  ];
 
-function index() {
-    return (
-        <div>
-            
-        </div>
-    )
+  const cardClick = index => {
```

Comment:
maybe instead of 
```javascript
 const cardClick = index => {
    if (selectedIndex === index) {
      setSelectedIndex(null);
    } else {
      setSelectedIndex(index);
    }
  };
```

do 

```javascript
 const cardClick = index => setSelectedIndex( index || null )
```